<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1001</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="GeneralPanel" name="generalPanel">
     <attribute name="title">
      <string>通用面板</string>
     </attribute>
    </widget>
    <widget class="SpeicalBossWidget" name="specialBossPanel">
     <attribute name="title">
      <string>角和无妄者</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="logPlainText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>791</width>
      <height>171</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="logLevelBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>540</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>debug</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>information</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>warning</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>critical</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="isBusyBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>600</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>脚本正在运行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>650</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>中止脚本运行</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GeneralPanel</class>
   <extends>QWidget</extends>
   <header location="global">generalpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpeicalBossWidget</class>
   <extends>QWidget</extends>
   <header location="global">speicalbosswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
